static.config.js

```diff
@@ -1,191 +1,163 @@
-
 import { GraphQLClient } from 'graphql-request';
+import { SUPPORTED_LANG_CODES } from 'js/i18n/constants';
 
 // QUERIES
 import allServicePagesQuery from 'js/queries/allServicePagesQuery';
 import allTopicPagesQuery from 'js/queries/allTopicPagesQuery';
 import allDepartmentPagesQuery from 'js/queries/allDepartmentPagesQuery';
 import topServicesQuery from 'js/queries/topServicesQuery';
+import allThemesQuery from 'js/queries/allThemesQuery';
+
+
+const createGraphQLClientsByLang = (lang) => {
```

Comment:
confused -- why are these changes showing in this PR? Wasn't this part of Ben's PR https://github.com/cityofaustin/janis/pull/206

Reply:
Mine was a PR to his PR

Reply:
OOOoooOOooOOOoOOOOOOoooOOo